src/components/Renderer/widgets/widget-util.tsx

```diff
@@ -28,11 +29,19 @@ export interface Widget {
 	displayName: string;
 }
 
+const DATE_FORMAT = 'MMM do yyyy';
+const TIME_FORMAT = 'h:mm a';
+
 export function formatTimestamp(timestamp: string, uiSchema: UiSchema = {}) {
-	const uiFormat = get(uiSchema, ['ui:options', 'dtFormat']);
-	return uiFormat ? format(new Date(timestamp), uiFormat) : timestamp;
+	const uiFormat =
+		get(uiSchema, ['ui:options', 'dtFormat']) ||
+		`${DATE_FORMAT}, ${TIME_FORMAT}`;
+	return format(new Date(timestamp), uiFormat);
```

Comment:
Hey @grahammcculloch, would that be still ok for you if I add a default format ?

Reply:
@JSReds yes - I think this is a good change and won't cause any issues with existing JF code.